_sections/inf2/inf2b.md

```diff
@@ -11,6 +11,7 @@ links:
 ---
 
 - Revision Hive Mind doc: [CLICK ME](https://drive.google.com/folderview?id=0BzxEStTA9O-SamF2YWFWRFJCNGc&usp=sharing)
+- [Revision Formulae for Learning Thread (pdf)](../../resources/inf2b/Inf2B%20Formulae.pdf) [(LaTex)](../../resources/inf2b/Inf2B%20Formulae.tex)
```

Comment:
You don't need the `../..` here, you can use `/resources/...`.

Reply:
I'd also maybe recommend removing the spaces in your file paths, or changing them to `_`.

Reply:
Will it mess things up if I leave it as is?

Reply:
For which point?

Reply:
The pathnames? Can I just change them or will it require reuploading and another pull request?

Reply:
You can change them on GitHub and it’ll add it to your pull request